bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -297,6 +297,23 @@
 				</with>
          	</visibleWhen>
          </command>
+         <command
+               commandId="com.espressif.idf.ui.installidfcomponents"
+               icon="icons/ESP-IDF Install Tools.png"
```

Comment:
We need to request a new icon, please add it to the icons request list in the wiki

Reply:
Requested by adding on the wiki page